nautobot/dcim/filters.py

```diff
@@ -333,11 +344,93 @@ class LocationFilterSet(NautobotFilterSet, StatusModelFilterSetMixin, TenancyFil
         field_name="location_type__content_types",
         choices=FeatureQuery("locations").get_choices,
     )
+    has_circuit_terminations = RelatedMembershipBooleanFilter(
```

Comment:
Should there be a `circuit_terminations` `NaturalKeyOrPKMultipleChoiceFilter` as well?

Reply:
Those are in Meta.fields just like Site model now, and they seemed to be mapped to ModelMultipleChoiceFilter.

Reply:
So I added `racks`, `clusters`, and `vlans` filters as `NaturalKeyOrPKMultipleChoiceFilter`. I am not too sure the other field for `NaturalKeyOrPKMultipleChoiceFilter` to map to for `circuit_terminations` and `prefixes`. Both models do not have slug or a sensible "NaturalKey" to map to.

Reply:
I think that makes sense for now, yeah. Thanks. :-)